go/types/opcache.go

```diff
@@ -28,12 +28,19 @@ type opCacheStore interface {
 type opCache interface {
 	MapSet(mapKey Value, mapVal Value)
```

Comment:
I debated whether or not to remove these. Each one can be replaced with a to it's corresponding Graph method like so:
MapSet(key, val)   => GraphMapSet(nil, key, val)

I'll take the approach that less api calls wins in this case against slightly more complex function signatures. I also deleted the corresponding, ListOp(), SetOp(), and Map() functions on opCacheIterator().